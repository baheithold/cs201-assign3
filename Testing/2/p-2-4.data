1 0 41 ; 3 1 367 ; 12 13 399 ; 3 1 491 ; 7 6 90 ; 
0 9 124 ; 2 7 124 ; 5 10 453 ; 9 1 364 ; 3 3 272 ; 
4 9 488 ; 5 0 93 ; 2 15 224 ; 4 7 269 ; 10 1 384 ; 
3 13 127 ; 11 10 52 ; 5 3 118 ; 7 15 468 ; 10 3 413 ; 
3 3 420 ; 4 5 372 ; 3 9 117 ; 10 10 0 ; 12 15 24 ; 
9 10 230 ; 3 6 10 ; 6 5 225 ; 5 9 70 ; 8 6 401 ; 
12 6 40 ; 1 11 27 ; 10 4 83 ; 5 0 255 ; 4 0 237 ; 
14 11 269 ; 5 9 475 ; 10 2 478 ; 3 0 437 ; 9 6 362 ; 
15 6 231 ; 10 9 327 ; 
