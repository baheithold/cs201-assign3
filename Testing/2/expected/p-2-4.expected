0: 1
1: 0(1)41 11(1)27
2: 5(0)93 10(11)52 14(11)269
3: 4(10)83 9(5)70
4: 3(9)117
5: 6(3)10 13(3)127
6: 7(6)90 8(6)401 12(6)40
7: 2(7)124 15(12)24
weight: 1568
